src/Faithlife.Utility/StringUtility.cs

```diff
@@ -78,6 +78,38 @@ public static string ReplaceOrdinal(this string source, string oldValue, string
 #endif
 		}
 
+#if NETSTANDARD2_0
+		/// <summary>
+		/// Calls string.Contains with StringComparison.Ordinal.
```

Comment:
This isn't true, based on the code below.

The comment should describe the purpose and output of the method, not its implementation. E.g., "Returns true if the string contains the specified character, using ordinal comparison."